### PR TITLE
Fixing release publishing

### LINKED ISSUE #619: Releases seem to be broken
It seems like the uploading github releases is not working.

### PR DESCRIPTION
**Pull request by mithro**
Fixes #619.

### CODE REVIEW

**Review by fangism — COMMENTED** [review]
Thanks for helping to fix this!

File: .github/workflows/success.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ bin)
     # Create a tag of form v0.0-183-gdf2b162-20191112132344
     rm -rf releasing/out
     git tag "$TAG" || true
+    echo "::set-output name=TAG::$(echo $TAG)"
```
Comment (umarcor) [inline .github/workflows/success.sh:37]: I suggest using `echo "TAG=$TAG" >> $GITHUB_ENV` instead. See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable.

Comment (mithro) [reply .github/workflows/success.sh:37]: Done!

**Comment by mithro** [comment]
@hzeller / @fangism - This should be ready to merge.

File: .github/workflows/success.sh (hunk 1 above)
Comment (mithro) [reply .github/workflows/success.sh:37]: This didn't seem to work - https://github.com/mithro/verible/actions/runs/403055934

**Comment by fangism** [comment]
> @hzeller / @fangism - This should be ready to merge.

Latest comment says 'not quite' :).  Ping me again when ready.

File: .github/workflows/success.sh (hunk 1 above)
Comment (umarcor) [reply .github/workflows/success.sh:37]: See https://github.com/mithro/verible/pull/2

**Comment by umarcor** [comment]
Note that tag 'master' and the corresponding release should be removed from the repo after this is merged.

**Comment by mithro** [comment]
@hzeller / @fangism - I think we should merge this and then @umarcor can send a pull request with the update.

**Comment by fangism** [comment]
> @hzeller / @fangism - I think we should merge this and then @umarcor can send a pull request with the update.

Ok, initiating merge/import...

**Comment by mithro** [comment]
Looks like after #625 was fixed, this change started working! https://github.com/google/verible/releases
Screenshot from 2020-12-08 12-34-57

**Pull request closed without merging**